### ISSUE TITLE
ASSA resolution crash...

### ISSUE THREAD

**Issue by Charvelx04** [SubtitleEdit/subtitleedit#5500]
I haven't run across this before, and keep in mind I am still on the beta you had me download from an issue before.

I have up'd the file I'm working with to DropAPK.. so if you need it, let me know.

It brings up this query window, then crashes.

![unknownASSAcrash](https://user-images.githubusercontent.com/35577096/142703517-c5698f09-761f-44fe-9cd3-b1294fc10597.png)

**Comment by Charvelx04** [comment]
Okay... an update to this.

I'm beginning to think all those crashes when trying to extract an ASS formatted sub from an MKV file, is the problem of a poorly encoded MKV. 

I tested this theory by downloading a different subtitle. I opened it first via SE, then opened video to match it. App crashed.
I renamed that same sub to the name of video, opened it first thru the app, it crashed.

I used Handbrake to re-encode to an MP4. No issues.

I tried again on a second computer that hasn't had SE installed, it too crashed when opening the MKV, but not the MP4.

So... not sure where you want to go with this.

**Comment by niksedk** [comment]
OK, could you email or post a link to the file?

**Comment by Charvelx04** [comment]
https://drop.download/gci8101g7pwv

**Comment by niksedk** [comment]
Thx for the file :)
Seems to work okay for me in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.3/SubtitleEditBeta.zip

Could you verify?

**Comment by Charvelx04** [comment]
Verified still crashes.

But... like the first time it crashed, I was using VLC. I switched over to MPC-HC, and no issues. 

Nature of the beast. MPC-HC literally opens it's own window when you open a sub with corresponding media. VLC does not.
VLC crashes with, what I am going to keep assuming is.. a poorly encoded MKV file with .ass subs.. whereas, MPC-HC does not.

Since I work more with MKV files than MP4s when it comes to sub corrections and submissions, I'll use MPC from here on out, and leave any VLC issues aside for now.
